src/lib/accessibility/Behaviors/Chat/chatBehavior.ts

```diff
@@ -21,7 +21,7 @@ const ChatBehavior: Accessibility = (props: any) => ({
     props: {
       shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.Enter,
       direction: FocusZoneDirection.vertical,
-      shouldHandleKeyDownCapture: false,
+      shouldResetActiveElementWhenTabFromZone: true,
```

Comment:
hmm, have you replicated all changes regarding FocusZone, together with types file? If the prop you're using is not on a list of handled props of a component, it might raise such error

Reply:
oh, you're right, missed to set it in PropTypes. Fixed, thank you!